soc/nxp/mcx/mcxn/soc.c

```diff
@@ -26,9 +26,9 @@ void soc_reset_hook(void)
 
 #endif
 
-#define FLEXCOMM_CHECK_2(n)	\
-	BUILD_ASSERT((DT_NODE_HAS_COMPAT(n, nxp_lpuart) == 0) &&		\
-		     (DT_NODE_HAS_COMPAT(n, nxp_lpi2c) == 0),			\
+#define FLEXCOMM_CHECK_2(n)                                                                        \
+	BUILD_ASSERT((DT_NODE_HAS_COMPAT(n, nxp_lpuart) == 0) &&                                   \
+			     (DT_NODE_HAS_COMPAT(n, nxp_lpi2c) == 0),                              \
```

Comment:
clang seemed to skew this line

Reply:
why?

Reply:
Don't know

Reply:
i think it's fine

Reply:
Me too. @nordicjm can you close your changes request or give your opinion about it.

Reply:
technically speaking, PR should focus on what it is trying to do and not also do "clean-up" that clang seems to be doing here. Okay to have a separate commit for the cleanup but makes it generally harder to review if you mix cleanup and actual changes.